CHANGELOG.md

```diff
@@ -2,11 +2,17 @@
 
 ## Unreleased
 
+## 0.10.0 (03-27-2021)
```

Comment:
nit: I like when the Changlog have hyperlinks to releases/changes, PRs
it is even sometimes helpful during development/bug fixing (at least for me 😉 )

Reply:
Me too but it's annoying to do after the fact. If we let this pass, I'll make sure we make every PR include link to itself in changelog in future :)

Reply:
Actually updated it. It was just a handful of changes.